pkgs/development/python-modules/cffi/default.nix

```diff
@@ -11,6 +11,44 @@ if isPyPy then null else buildPythonPackage rec {
     sha256 = "sha256-1AC/uaN7E1ElPLQCZxzqfom97MKU6AFqcH9tHYrJNPk=";
   };
 
+  patches = [
+    #
```

Comment:
Can you try to condense this comment a bit?

Reply:
Thank you for reviewing.

This part could be condensed to remove details:

```
# It uses a more sophisticated strategy for creating trampolines to support
# closures that works on Apple Silicon, while the simple approach that cffi
# falls back on does not, so this patch enables code that uses closures on
# M1 Macs again.
```

This part can be removed if we take care of unbreaking pyOpenSSL right after this:

```
# Notably, pyOpenSSL is impacted and will be fixed by this, reported in
#
#   https://github.com/pyca/pyopenssl/issues/873
```

Do you have any other suggestions? It's hard for me to evaluate what can be removed or condensed, because I am the original author and have thought about the details of this issue too much.

Reply:
All the info sounds like a good candidate for the commit message rather than a code comment.

Reply:
Moved most of the information contained in the comment into the commit message.